spec/integration/scenarios_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe "Scenarios" do
+  let(:page_1) { build :page }
+  let(:page_2) { build :page }
+  let(:page_3) { build :page }
+  let(:event_1) { build :event_render_partial }
+  let(:event_2) { build :event_render_partial }
+  let(:event_3) { build :event_render_partial }
+  let(:event_4) { build :event_render_partial }
+  let(:event_5) { build :event_query }
+  let(:event_6) { build :event_query }
+
+  before do
+    page_1.events << event_1 << event_5 << event_2
+
+    page_2.events << event_3 << event_6 << event_4
+
+    page_3.events << event_5 << event_1 << event_2
+  end
+
+  it 'page hashes should be eq' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.